src/fmu/sumo/sim2sumo/_special_treatments.py

```diff
@@ -193,7 +193,8 @@ def vfp_to_arrow_dict(datafile, options):
         tuple: vfp keyword, then dictionary with key: table_name, value: table
     """
     logger = logging.getLogger(__file__ + ".vfp_to_arrow_dict")
-    resdatafiles = res2df.ResdataFiles(datafile)
+    filepath_no_suffix = Path(datafile).with_suffix("")
```

Comment:
Use `Path(datafile).stem()` instead?

Reply:
`.stem` returns only the filename, removing all other path info.
`datafile` here can be a full path e.g. `/folder/subfolder/file.suffix`.
`.stem` returns only `file`, but we need `/folder/subfolder/file`